src/tests.c

```diff
@@ -86,6 +86,7 @@ void random_group_element_test(secp256k1_ge *ge) {
             break;
         }
     } while(1);
+    ge->infinity = 0;
```

Comment:
Oh, on a second thought, this is not just necessary. `secp256k1_ge_set_xo_var` (a few lines above) sets `infinity` to 0.

I was playing around with multiple additional `VERIFY_CHECK`s when reviewing the schnorrsig PR and I wrongly believed that I found this bug ... 

Well whatever, the correct was correct before this PR and it's correct after this PR. If we want, we can reverse this commit in a later PR.